packages/forms/src/elements/tiles/Tiles.tsx

```diff
@@ -23,13 +23,13 @@ import { Icon } from './components/Icon';
 import { Label } from './components/Label';
 
 interface ITilesProps extends HTMLAttributes<HTMLDivElement> {
-  /** Value of the selected radio button */
+  /** Sets the value of the selected radio button */
   value?: string;
-  /** Callback when a radio is selected */
+  /** Handles radio selection */
   onChange?: ChangeEventHandler<HTMLInputElement>;
```

Comment:
@austingreendev I hadn't spotted this `ChangeEventHandler` before. I suspect this is the signature all Garden component `onChange` events should take. And, as documented here by @mikaelagurney, I think it is reasonable to leave the signature implicit (no extra `@param` JSDoc) since the React construct is pervasive and obvious.

Thoughts?